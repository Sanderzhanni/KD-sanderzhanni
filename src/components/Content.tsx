import React from "react";
import {Link} from "react-router-dom";
import {stories} from "../mocks/stories";

const Content = (): React.ReactElement =>(
  <>
    {stories.map((story, i) =>
      <div key={i} className={"story"}>
        <h3>{story.title}</h3>
        <p>
          {story.text}
        </p>
        <Link to={"TODO"}>Loe lisaks</Link>
      </div>
    )}
  </>
);

export default Content;