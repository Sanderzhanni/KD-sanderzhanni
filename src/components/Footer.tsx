import React from 'react';
import {Accordion, Card, Button} from "react-bootstrap";
import {Link} from 'react-router-dom';
import {FiPlus, FiMinus} from 'react-icons/fi';
import {TiSocialFacebookCircular} from 'react-icons/ti';
import {FaInstagram, FaSoundcloud} from 'react-icons/fa';

interface Props {
    isOpen: boolean;
    activeEvent: string
}

class Footer extends React.PureComponent<{}, Props>{

    constructor(props: {}) {
        super(props);
        this.state ={
            isOpen: false,
            activeEvent: "",
        }
    }

    //Changes the icon inside the accordion header accordingly
    private HandleAccordionToggle = (name: string) =>{
        //if its already active and user collapses it
        if(this.state.activeEvent === name){
            this.setState({
                activeEvent: "none"
            });
            return
        }
        //if use click on the button and it isn't yet active
        this.setState({
            activeEvent: name
        })
    };

    public render(){
        return(
            <>
                <div className={"footer-container"}>
                    <Acc HandleAccordionToggle={this.HandleAccordionToggle} activeName={this.state.activeEvent}/>
                    <SocialMediaIcons/>
                    <Disclaimer/>
                </div>
            </>
        )
    }

}

const Disclaimer = () => (
    <div className={"footer__disclaimer"}>
        <p>
            © 2001–2020 AS LHV Group, Tartu mnt 2, 10145, Tallinn. Oled finantsteenuseid pakkuvate ettevõtete AS LHV Pank, AS LHV Finance ja AS LHV Varahaldus veebilehel. Ettevõtetel AS LHV Pank, AS LHV Finance ja AS LHV Varahaldus on õigus salvestada kõiki neile sidevahendite teel antud korraldusi ja muud suhtlust ning kasutada vastavaid salvestisi antud korralduste või muude toimingute tõendamiseks. Enne finantsteenuse lepingu sõlmimist tutvu
        </p>
    </div>
);

const SocialMediaIcons = (): React.ReactElement =>(
  <>
      <div className={"socialmedia__icons__container"}>
          <Link to={"todo"}>
              <TiSocialFacebookCircular className={"socialmedia__icons"}/>
          </Link>
          <Link to={"todo"}>
              <FaInstagram className={"socialmedia__icons"}/>
          </Link>
          <Link to={"todo"}>
              <FaSoundcloud className={"socialmedia__icons"}/>
          </Link>
      </div>
  </>
);

interface AccProps {
    HandleAccordionToggle: Function;
    activeName: string;

}

const Acc = (props: AccProps): React.ReactElement =>(
  <>
      <Accordion>
          <AccordionCard name={"TULE KLIENDIKS"}
                         eventKey={"0"}
                         HandleAccordionToggle={props.HandleAccordionToggle}
                         activeName={props.activeName}
          >
              <FooterLink path={"todo"} name={"Eraklient"}/>
              <FooterLink path={"todo"} name={"Äriklient"}/>
              <FooterLink path={"todo"} name={"Noortepank"}/>
              <FooterLink path={"todo"} name={"Privaatpangandus"}/>
              <FooterLink path={"todo"} name={"Ava konto"}/>
          </AccordionCard>
          <AccordionCard name={"LEIA KIIRELT"} eventKey={"1"} HandleAccordionToggle={props.HandleAccordionToggle} activeName={props.activeName}>
              <FooterLink path={"todo"} name={"Hinnakiri ja tingimused"}/>
              <FooterLink path={"todo"} name={"Ettepanekud ja kaebused"}/>
              <FooterLink path={"todo"} name={"Turvalisus"}/>
              <FooterLink path={"todo"} name={"Teata petulehest"}/>
              <FooterLink path={"todo"} name={"KKK"}/>
          </AccordionCard>
          <AccordionCard name={"LHV"} eventKey={"3"} HandleAccordionToggle={props.HandleAccordionToggle} activeName={props.activeName}>
              <FooterLink path={"todo"} name={"Ettevõtest"}/>
              <FooterLink path={"todo"} name={"Tööpakkumised"}/>
              <FooterLink path={"todo"} name={"Uudised"}/>
              <FooterLink path={"todo"} name={"Investor"}/>
              <FooterLink path={"todo"} name={"Finantsportaal"}/>
          </AccordionCard>
          <AccordionCard name={"KONTAKT"} eventKey={"4"} HandleAccordionToggle={props.HandleAccordionToggle} activeName={props.activeName}>
              <FooterLink path={"todo"} name={"Kontaktid"}/>
              <FooterLink path={"todo"} name={"Kontorid"}/>
              <FooterLink path={"todo"} name={"Broneeri kohtumisaeg"}/>
              <FooterLink path={"todo"} name={"Sularahaautomaadid"}/>
          </AccordionCard>
      </Accordion>
  </>
);

interface AccordionCardProps {
    children?:  React.ReactNode;
    name: string;
    eventKey: string;
    HandleAccordionToggle: any;
    activeName: string;
}

const AccordionCard = (props: AccordionCardProps) =>(
    <>
        <Card>
            <Card.Header>
                <Accordion.Toggle as={Button} variant="link" eventKey={props.eventKey} onClick={ () => props.HandleAccordionToggle(props.name)}>
                    <h3 className={"accordion-header"}>{props.name}</h3>
                    {props.activeName === props.name ? <FiMinus className={"accordion__icon"}/>: <FiPlus className={"accordion__icon"}/>}
                </Accordion.Toggle>
            </Card.Header>
            <Accordion.Collapse eventKey={props.eventKey}>
                <Card.Body>
                    {props.children}
                </Card.Body>
            </Accordion.Collapse>
        </Card>
    </>
);

interface FooterLinkProps {
    name: string;
    path: string;
}

const FooterLink = (props: FooterLinkProps) =>(
  <>
      <li>sth</li>
      <div className={"footer__body--link"}>
          <Link to={props.path}>
              <div>
                  {props.name}
              </div>
          </Link>
      </div>
  </>
);

export default Footer

