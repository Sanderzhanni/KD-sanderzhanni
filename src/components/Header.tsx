import React from 'react';


const Header = (): React.ReactElement =>(
  <>
    <div className={"nav-container"}>
      <div className={"nav"}>
        <div>
          <img src={process.env.PUBLIC_URL + 'img/lhv.png'} alt="lhv logo" className={"lhv-logo"}/>
        </div>
        <div className={"nav-button"}>Sisene</div>
      </div>
    </div>
  </>
);

export default Header;

