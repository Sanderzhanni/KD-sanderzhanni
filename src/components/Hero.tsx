import React from 'react';
import Carousel from "react-bootstrap/Carousel";

class Hero extends React.PureComponent{
  render(): React.ReactElement{
    return(
      <>
        <CarouselComp/>
      </>
    );
  }
}


const CarouselComp = (): React.ReactElement =>(
  <>
    <Carousel controls={false} className={"carousel__comp"}>
      <Carousel.Item>
        {/*image from the public/img directory*/}
        <img
          className="d-block w-100 --carousel__img"
          src={'https://www.lhv.ee/assets/images/headers/palkyle-m.svg?v=1583498979'}
          alt="slide"
        />
        <Carousel.Caption className={"carousel__caption"}>
          <h3 className={"caption__title"}>Võida topeltpalk</h3>
          <p>Too oma palk LHV-sse ja osale </p>
          <p>Loe lähemalt</p>
        </Carousel.Caption>
      </Carousel.Item>
      <Carousel.Item>
        <img
          className="d-block w-100 --carousel__img"
          src={"https://www.lhv.ee/assets/images/headers/mobile/vaartpaberikonto-m.svg"}
          alt="slide"
        />
        <Carousel.Caption className={"carousel__caption"}>
          <h3 className={"caption__title"}>Investeerimine = LHV</h3>
          <p>Balti aktsiatega kauplemine ja hoidmine LHV-s nüüd täiesti tasuta</p>
          <p>Loe lähemalt</p>
        </Carousel.Caption>
      </Carousel.Item>
      <Carousel.Item>
        <img
          className="d-block w-100 --carousel__img"
          src={"https://www.lhv.ee/assets/images/headers/pension-m-et.svg?v=1583498979"}
          alt="slide"
        />
        <Carousel.Caption className={"carousel__caption --color__black --position__low"}>
          <li className={"caption__li"}>ajalooliselt kõrge pikaajaline tootlus</li>
          <li className={"caption__li"}>investeerime ka sinna, kuhu sa ise ei saa</li>
          <li className={"caption__li"}>aitad kaasa Eesti majanduskasv</li>
          <p className={"caption__redir"}>Loe lähemalt</p>
        </Carousel.Caption>
      </Carousel.Item>
    </Carousel>
  </>
);

export default Hero;