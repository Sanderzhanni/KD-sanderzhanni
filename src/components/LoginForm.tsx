import *  as React from 'react';
import classNames from "classnames";

const LoginForm = (): React.ReactElement => {
  const [ selected, setSelected ] = React.useState(3);

  const select = (number: number): void => {
    setSelected(number);
  };
  return (
    <>
      <div className="login-form">
        <ul>
          {/*not the correct way to select elements, but its fine here*/}
          <li onClick={(): void => select(1)}>
            <a href="javascript:void(0)" className={classNames("border-start", {selected: selected === 1})}>ID-kaart</a>
          </li>
          <li onClick={(): void => select(2)}>
            <a href="javascript:void(0)" className={classNames({selected: selected === 2})}>Mobiil-ID</a>
          </li>
          <li onClick={(): void => select(3)}>
            <a href="javascript:void(0)" className={classNames({selected: selected === 3})}>Smart-ID</a>
          </li>
          <li onClick={(): void => select(4)}>
            <a href="javascript:void(0)" className={classNames({selected: selected === 4})}>PIN-kalkulaator</a>
          </li>
          <li onClick={(): void => select(5)}>
            <a href="javascript:void(0)" className={classNames("border-end", {selected: selected === 5})}>Salas√µna</a>
          </li>
        </ul>
        <div className={"username"}>
          <input type="text" placeholder={"Kasutajanimi"}/>
        </div>
        <div className="login-btn">
          <button>Sisene</button>
        </div>
      </div>
    </>
  );
};

export default LoginForm;