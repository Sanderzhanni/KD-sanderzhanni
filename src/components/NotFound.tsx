import * as React from 'react';

const NotFound = (): React.ReactElement => {
  return(
    <>
      <div className="not-found">
        THIS PATH HAS NOT BEEN IMPLEMENTED YET
      </div>
    </>
  );
};

export default NotFound;