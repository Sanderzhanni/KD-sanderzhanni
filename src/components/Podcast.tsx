import React from "react";
import {podcasts} from "../mocks/podcasts";
import {Link} from "react-router-dom";

const Podcast = (): React.ReactElement =>(
  <>
    <div  className={"podcast"}>
      <h3>Kuula LHV Podcasti</h3>
      {podcasts.map((podcast: string, i) =>
        <div key={i}>
          {podcast}
        </div>
      )}
      <Link to={"TODO"}>Kuula k√µiki</Link>
    </div>
  </>
);

export default Podcast;