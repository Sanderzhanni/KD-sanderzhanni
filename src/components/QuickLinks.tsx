import * as React from 'react';

const QuickLinks = (): React.ReactElement => {
  return(
    <>
      <div className="quick-links">
        <a href="TODO">IGAPÃ„EVAPANGANDUS</a>
        <a href="TODO">RAHA KASVATAMINE</a>
        <a href="TODO">LAEN JA LIISIN</a>
        <a href="TODO">PENSION</a>
        <a href="TODO">VAJAD ABI?</a>
      </div>
    </>
  );
};

export default QuickLinks;