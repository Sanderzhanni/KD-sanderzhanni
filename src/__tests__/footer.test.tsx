import React from 'react';
import Footer from "../components/Footer";
import {FooterLink, AccordionCard} from "../components/Footer";
import {Accordion} from "react-bootstrap";
import renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";
import Enzyme, {mount, shallow} from "enzyme";
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

describe("testing footer component", () =>{
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Footer/>
        </BrowserRouter>
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("should change state value", ()=>{
    const wrapper = mount(
      <BrowserRouter>
        <Footer />
      </BrowserRouter>
    );
    const component = wrapper.find(Footer);
    const target = component.find(".accordion-header").first();
    expect(component.state('activeEvent')).toBe("");
    target.simulate('click');
    expect(component.state('activeEvent')).toBe(target.text());
  });

  it("Accordion", () =>{
    const name = "test";
    const eventKey = "0";
    const wrapper = shallow(<AccordionCard HandleAccordionToggle={(): null => null} activeName={""} eventKey={eventKey} name={name}/>);
    const nameH3 = wrapper.find('h3');
    const toggle = wrapper.find(Accordion.Toggle);
    const collapse = wrapper.find(Accordion.Collapse);
    expect(toggle.prop('eventKey')).toBe(eventKey);
    expect(collapse.prop('eventKey')).toBe(eventKey);
    expect(nameH3.text()).toBe(name);
  });

  it("FooterLink", () =>{
    const name = "test";
    const path = "/testpath";
    const wrapper = shallow(<FooterLink name={name} path={path}/>);
    const nameDiv = wrapper.find('.footerlink__name');
    const link = wrapper.find('Link');
    expect(link.prop('to')).toBe('/testpath');
    expect(nameDiv.text()).toBe(name);
  });
});