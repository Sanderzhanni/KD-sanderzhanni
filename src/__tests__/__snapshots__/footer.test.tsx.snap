// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing footer component renders correctly 1`] = `
<div
  className="footer-container"
>
  <div
    className="accordion"
  >
    <div
      className="card"
    >
      <div
        className="card-header"
      >
        <button
          className="btn btn-link"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          <h3
            className="accordion-header"
          >
            TULE KLIENDIKS
          </h3>
          <svg
            className="accordion__icon"
            fill="none"
            height="1em"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="2"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <line
              x1="12"
              x2="12"
              y1="5"
              y2="19"
            />
            <line
              x1="5"
              x2="19"
              y1="12"
              y2="12"
            />
          </svg>
        </button>
      </div>
      <div
        aria-expanded={null}
        className="collapse"
      >
        <div
          className="card-body"
        >
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Eraklient
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Äriklient
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Noortepank
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Privaatpangandus
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Ava konto
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-header"
      >
        <button
          className="btn btn-link"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          <h3
            className="accordion-header"
          >
            LEIA KIIRELT
          </h3>
          <svg
            className="accordion__icon"
            fill="none"
            height="1em"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="2"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <line
              x1="12"
              x2="12"
              y1="5"
              y2="19"
            />
            <line
              x1="5"
              x2="19"
              y1="12"
              y2="12"
            />
          </svg>
        </button>
      </div>
      <div
        aria-expanded={null}
        className="collapse"
      >
        <div
          className="card-body"
        >
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Hinnakiri ja tingimused
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Ettepanekud ja kaebused
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Turvalisus
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Teata petulehest
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                KKK
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-header"
      >
        <button
          className="btn btn-link"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          <h3
            className="accordion-header"
          >
            LHV
          </h3>
          <svg
            className="accordion__icon"
            fill="none"
            height="1em"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="2"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <line
              x1="12"
              x2="12"
              y1="5"
              y2="19"
            />
            <line
              x1="5"
              x2="19"
              y1="12"
              y2="12"
            />
          </svg>
        </button>
      </div>
      <div
        aria-expanded={null}
        className="collapse"
      >
        <div
          className="card-body"
        >
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Ettevõtest
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Tööpakkumised
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Uudised
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Investor
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Finantsportaal
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-header"
      >
        <button
          className="btn btn-link"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          <h3
            className="accordion-header"
          >
            KONTAKT
          </h3>
          <svg
            className="accordion__icon"
            fill="none"
            height="1em"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="2"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <line
              x1="12"
              x2="12"
              y1="5"
              y2="19"
            />
            <line
              x1="5"
              x2="19"
              y1="12"
              y2="12"
            />
          </svg>
        </button>
      </div>
      <div
        aria-expanded={null}
        className="collapse"
      >
        <div
          className="card-body"
        >
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Kontaktid
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Kontorid
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Broneeri kohtumisaeg
              </div>
            </a>
          </div>
          <div
            className="footer__body--link"
          >
            <a
              href="/todo"
              onClick={[Function]}
            >
              <div
                className="footerlink__name"
              >
                Sularahaautomaadid
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="socialmedia__icons__container"
  >
    <a
      href="/todo"
      onClick={[Function]}
    >
      <svg
        baseProfile="tiny"
        className="socialmedia__icons"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        version="1.2"
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M18.354 5.624c-1.75-1.741-3.888-2.624-6.354-2.624-2.489 0-4.633.884-6.373 2.625-1.743 1.741-2.627 3.887-2.627 6.375 0 2.465.883 4.603 2.624 6.354 1.741 1.756 3.886 2.646 6.376 2.646 2.467 0 4.605-.89 6.356-2.643 1.755-1.753 2.644-3.892 2.644-6.357 0-2.488-.89-4.634-2.646-6.376zm-1.412 11.319c-1.137 1.139-2.436 1.788-3.942 1.985v-4.928h2v-2h-2v-1.4c0-.331.269-.6.601-.6h1.399v-2h-1.397c-.742 0-1.361.273-1.857.822-.496.547-.746 1.215-.746 2.008v1.17h-2v2h2v4.93c-1.522-.195-2.826-.845-3.957-1.984-1.375-1.384-2.043-3.002-2.043-4.946 0-1.966.667-3.588 2.042-4.96 1.37-1.373 2.992-2.04 4.958-2.04 1.945 0 3.562.668 4.945 2.043 1.383 1.372 2.055 2.994 2.055 4.957 0 1.941-.673 3.559-2.058 4.943z"
        />
      </svg>
    </a>
    <a
      href="/todo"
      onClick={[Function]}
    >
      <svg
        className="socialmedia__icons"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
        />
      </svg>
    </a>
    <a
      href="/todo"
      onClick={[Function]}
    >
      <svg
        className="socialmedia__icons"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 640 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M111.4 256.3l5.8 65-5.8 68.3c-.3 2.5-2.2 4.4-4.4 4.4s-4.2-1.9-4.2-4.4l-5.6-68.3 5.6-65c0-2.2 1.9-4.2 4.2-4.2 2.2 0 4.1 2 4.4 4.2zm21.4-45.6c-2.8 0-4.7 2.2-5 5l-5 105.6 5 68.3c.3 2.8 2.2 5 5 5 2.5 0 4.7-2.2 4.7-5l5.8-68.3-5.8-105.6c0-2.8-2.2-5-4.7-5zm25.5-24.1c-3.1 0-5.3 2.2-5.6 5.3l-4.4 130 4.4 67.8c.3 3.1 2.5 5.3 5.6 5.3 2.8 0 5.3-2.2 5.3-5.3l5.3-67.8-5.3-130c0-3.1-2.5-5.3-5.3-5.3zM7.2 283.2c-1.4 0-2.2 1.1-2.5 2.5L0 321.3l4.7 35c.3 1.4 1.1 2.5 2.5 2.5s2.2-1.1 2.5-2.5l5.6-35-5.6-35.6c-.3-1.4-1.1-2.5-2.5-2.5zm23.6-21.9c-1.4 0-2.5 1.1-2.5 2.5l-6.4 57.5 6.4 56.1c0 1.7 1.1 2.8 2.5 2.8s2.5-1.1 2.8-2.5l7.2-56.4-7.2-57.5c-.3-1.4-1.4-2.5-2.8-2.5zm25.3-11.4c-1.7 0-3.1 1.4-3.3 3.3L47 321.3l5.8 65.8c.3 1.7 1.7 3.1 3.3 3.1 1.7 0 3.1-1.4 3.1-3.1l6.9-65.8-6.9-68.1c0-1.9-1.4-3.3-3.1-3.3zm25.3-2.2c-1.9 0-3.6 1.4-3.6 3.6l-5.8 70 5.8 67.8c0 2.2 1.7 3.6 3.6 3.6s3.6-1.4 3.9-3.6l6.4-67.8-6.4-70c-.3-2.2-2-3.6-3.9-3.6zm241.4-110.9c-1.1-.8-2.8-1.4-4.2-1.4-2.2 0-4.2.8-5.6 1.9-1.9 1.7-3.1 4.2-3.3 6.7v.8l-3.3 176.7 1.7 32.5 1.7 31.7c.3 4.7 4.2 8.6 8.9 8.6s8.6-3.9 8.6-8.6l3.9-64.2-3.9-177.5c-.4-3-2-5.8-4.5-7.2zm-26.7 15.3c-1.4-.8-2.8-1.4-4.4-1.4s-3.1.6-4.4 1.4c-2.2 1.4-3.6 3.9-3.6 6.7l-.3 1.7-2.8 160.8s0 .3 3.1 65.6v.3c0 1.7.6 3.3 1.7 4.7 1.7 1.9 3.9 3.1 6.4 3.1 2.2 0 4.2-1.1 5.6-2.5 1.7-1.4 2.5-3.3 2.5-5.6l.3-6.7 3.1-58.6-3.3-162.8c-.3-2.8-1.7-5.3-3.9-6.7zm-111.4 22.5c-3.1 0-5.8 2.8-5.8 6.1l-4.4 140.6 4.4 67.2c.3 3.3 2.8 5.8 5.8 5.8 3.3 0 5.8-2.5 6.1-5.8l5-67.2-5-140.6c-.2-3.3-2.7-6.1-6.1-6.1zm376.7 62.8c-10.8 0-21.1 2.2-30.6 6.1-6.4-70.8-65.8-126.4-138.3-126.4-17.8 0-35 3.3-50.3 9.4-6.1 2.2-7.8 4.4-7.8 9.2v249.7c0 5 3.9 8.6 8.6 9.2h218.3c43.3 0 78.6-35 78.6-78.3.1-43.6-35.2-78.9-78.5-78.9zm-296.7-60.3c-4.2 0-7.5 3.3-7.8 7.8l-3.3 136.7 3.3 65.6c.3 4.2 3.6 7.5 7.8 7.5 4.2 0 7.5-3.3 7.5-7.5l3.9-65.6-3.9-136.7c-.3-4.5-3.3-7.8-7.5-7.8zm-53.6-7.8c-3.3 0-6.4 3.1-6.4 6.7l-3.9 145.3 3.9 66.9c.3 3.6 3.1 6.4 6.4 6.4 3.6 0 6.4-2.8 6.7-6.4l4.4-66.9-4.4-145.3c-.3-3.6-3.1-6.7-6.7-6.7zm26.7 3.4c-3.9 0-6.9 3.1-6.9 6.9L227 321.3l3.9 66.4c.3 3.9 3.1 6.9 6.9 6.9s6.9-3.1 6.9-6.9l4.2-66.4-4.2-141.7c0-3.9-3-6.9-6.9-6.9z"
        />
      </svg>
    </a>
  </div>
  <div
    className="footer__disclaimer"
  >
    <p>
      © 2001–2020 AS LHV Group, Tartu mnt 2, 10145, Tallinn. Oled finantsteenuseid pakkuvate ettevõtete AS LHV Pank, AS LHV Finance ja AS LHV Varahaldus veebilehel. Ettevõtetel AS LHV Pank, AS LHV Finance ja AS LHV Varahaldus on õigus salvestada kõiki neile sidevahendite teel antud korraldusi ja muud suhtlust ning kasutada vastavaid salvestisi antud korralduste või muude toimingute tõendamiseks. Enne finantsteenuse lepingu sõlmimist tutvu
    </p>
  </div>
</div>
`;
